src/connector/src/sink/encoder/mod.rs

```diff
@@ -150,6 +150,33 @@ pub enum CustomJsonType {
     None,
 }
 
+/// How the jsonb type is encoded.
+///
+/// - `string`: encode jsonb as string. `[1, true, "foo"] -> "[1, true, \"foo\"]"`
+/// - `object`: encode jsonb as json object. `[1, true, "foo"] -> [1, true, "foo"]`
```

Comment:
This is not accurate. `[1, true, "foo"]` is a json **array** rather than a json **object**. This is why the original issue calls it **dynamic** type: it may be one of json `null` / `boolean` / `number` / `string` / `array` / `object`.

Reply:
So rename it to dynamic?

Reply:
Yes

Reply:
Okay. I use `object` bc the function name is `datum_to_json_object()`
`

Reply:
Because the top level json returned by this function is always an object containing multiple fields, corresponding to a table row containing multiple columns.